src/lib/y2partitioner/actions/controllers/filesystem.rb

```diff
@@ -344,6 +348,17 @@ def finish
           @restorer.update_checkpoint
         end
 
+        # Removes from the block device or its encryption layer a LvmPv not associated to an LvmVg
+        # (bsc#1129663)
+        def remove_unused_lvm_pv
+          device = blk_device.encryption ? blk_device.encryption : blk_device
+          lvm_pv = device.lvm_pv
+
+          return unless device.lvm_pv
+
+          device.remove_descendants if lvm_pv.descendants.empty?
```

Comment:
NP:
```suggestion
          device = blk_device.encryption || blk_device
          lvm_pv = device.lvm_pv

          device.remove_descendants if lvm_pv && lvm_pv.descendants.none?
```

Reply:
> evice = blk_device.encryption || blk_device

Nice. Believe or not, I was thinking for a while: "David, there is something wrong with that assignation using a ternary operator"

But then I forgot to change it. 

Thanks!